src/v/pandaproxy/schema_registry/handlers.cc

```diff
@@ -146,7 +146,7 @@ put_config_subject(server::request_t rq, server::reply_t rp) {
     // in cache yet
     co_await get_or_load(rq, [&rq, sub, config]() -> ss::future<bool> {
         // Throw if it's not found
-        co_await rq.service().schema_store().get_compatibility(sub);
+        // co_await rq.service().schema_store().get_compatibility(sub);
```

Comment:
`// ?`

maybe delete dead code (or add a TODO?)

Reply:
> `// ?`
> 
> maybe delete dead code (or add a TODO?)

Changing it to:
```cpp
    // Ensure we see latest writes
    co_await rq.service().writer().read_sync();
    co_await rq.service().writer().write_config(sub, config.compat);
```